src/components/Finishers-Components/Finisher-Member.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <img :src="require('@/assets/images/' + finisher.finisherImage + '')" alt="finishermage">
+    <h5>{{finisher.finisherName}}</h5>
+    <p>{{finisher.finisherDate}}</p>
+    <p>{{finisher.finisherImage}}</p>
```

Comment:
I think you forgot to remove the filename of the photo here

Reply:
hi, im not sure what you mean by the filename of the photo here?

Reply:
ah lol nvm found it whahhahaha nani